extensions/test-extension/src/custom-context-menu/codingValues.ts

```diff
@@ -23,28 +23,44 @@ const codingValues = {
   'SCT:69536005': {
     text: 'Head',
     type: 'site',
+    style: {
+      color: 'red',
```

Comment:
Can we normalize this to RGB or hex instead of text? What shade of red are we referring to, and what is its brightness?